src/flownet/templates/ahm_config.ert.jinja2

```diff
@@ -3,27 +3,37 @@
 INSTALL_JOB DELETE_IN_CURRENT_ITERATION ./DELETE_IN_CURRENT_ITERATION
 
 REFCASE <CONFIG_PATH>/SYNTHETIC_REFCASE
+{%- if ert_config.analysis: %}
+DEFINE <CONFIG_FILE>  ../brugge_parameters.yml
+{%- endif %}
 DEFINE <RUN_PATH> {{ ert_config.runpath }}
 DEFINE <ECL_BASE> {{ ert_config.eclbase }}
 DEFINE <RANDOM_SAMPLES> ./parameters.json
-DEFINE <YAML_OBS> {{ ert_config.yamlobs }}
+{%- if ert_config.analysis: %}
 DEFINE <ANALYSIS_METRIC> {{ ert_config.analysis.metric }}
 DEFINE <ANALYSIS_QUANTITY> {{ ert_config.analysis.quantity }}
 DEFINE <ANALYSIS_START> {{ ert_config.analysis.start }}
 DEFINE <ANALYSIS_END> {{ ert_config.analysis.end }}
 DEFINE <ANALYSIS_OUTFILE> {{ ert_config.analysis.outfile }}
+{%- endif %}
```

Comment:
Do we really need to define all the parameters here? Can't that just come from the config file without further cluttering in the ahm_config file?

Reply:
Good suggestion. We now pass the config file as input to the workflow, and all those inputs are read from there